tools/add_triage_signature.py

```diff
@@ -478,7 +478,14 @@ def __init__(self, *args, **kwargs):
 
     def _process_ticket(self, url, issue_key):
         md = get_metadata_json(url)
-        installconfig = get_installconfig_yaml(url)
+
+        try:
+            installconfig = get_installconfig_yaml(url)
+        except FailedToGetInstallConfigException:
+            self._update_triaging_ticket(
+                "Installation status signature cannot be created. This cluster's collected logs are missing install-config.yaml for an unknown reason, why is it missing?"
+            )
```

Comment:
maybe we should log the error as well
it will still not be easy to find the root-cause maybe (having to look throughout a couple of jenkins jobs logs) but at least we'll have it written somewhere

Reply:
Done